app/models/produce_item.rb

```diff
@@ -6,7 +6,6 @@ class ProduceItem < ApplicationRecord
 
   has_many :item_seasons
 
-  validates :name, presence: true,
-                   uniqueness: true
+  validates :name, presence: true, uniqueness: true
```

Comment:
This a comment that should be handled in a different PR. However it is a good opportunity to talk about uniqueness validation in a Rails app.

`uniqueness` is one of the Rails built-in validations that we should avoid. We can replace it for unique indices instead.

The reasoning behind this is that `uniqueness` does 1 request to the database to check if a record already exists (something like `SELECT 1 FROM produce_items WHERE name = 'an specific name'`). Unique indices, instead, are handled internally by the database, which makes things way faster. In small scale application this does not make much difference, but also there is no reason why we should not apply it.

Now, how do we even do that?

1. We remove the uniqueness validation
2. We add a migration to include unique indices: 
```rb
add_index :produce_items, :name, unique: true
```

We should that soon, because once there are duplicates in the database, things get way harder.

Reply:
so, for now we merge, and we do a new branch/pr with this migration?